tsconfig.json

```diff
@@ -11,6 +11,8 @@
     "checkJs": false,
     // Disallow features that require cross-file information for emit as we're using Babel
     "isolatedModules": true,
+    "strict": true,
```

Comment:
We'd better do it right from the very beginning :-)

Reply:
The `"extends": "@tsconfig/node16/tsconfig.json",` above doe this for you :)

Reply:
Oh :-) I totally missed the `extends` clause :-) I only looked at `compilerOptions` 😁